hrm/settings.py

```diff
@@ -78,9 +80,9 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'department',
-        'USER' : 'ceo',
-        'PASSWORD' : 'canton',
-        'HOST': '127.0.0.1',
+        'USER': 'ceo',
```

Comment:
That is not a good practice to store any credentials in repository - let's move them to environment variables and describe how to setup them in readme file.